content/docs/reference-glossary.md

```diff
@@ -1,43 +1,43 @@
+सिंगल पेज एप्लिकेशन एक ऐसा एप्लिकेशन है जो एकल HTML पेज को लोड करता है और आवश्यक सभी संपत्तियो को भी (जैसे जावास्क्रिप्ट और सीएसएस)जो एप्लिकेशन को चलाने के लिए आवश्यक है। पृष्ठ या बाद के पृष्ठों के साथ किसी भी बातचीत को सर्वर के लिए एक गोल यात्रा की आवश्यकता नहीं होती है जिसका अर्थ है कि पृष्ठ पुनः लोड नहीं किया गया है।
 
-Though you may build a single-page application in React, it is not a requirement. React can also be used for enhancing small parts of existing websites with additional interactivity. Code written in React can coexist peacefully with markup rendered on the server by something like PHP, or with other client-side libraries. In fact, this is exactly how React is being used at Facebook.
+यद्यपि आप React में सिंगल पेज एप्लिकेशन का निर्माण कर सकते हैं, यह कोई आवश्यकता नहीं है। React का उपयोग मौजूदा वेबसाइटों के छोटे हिस्सों को अतिरिक्त अन्तरक्रियाशीलता के साथ बढ़ाने के लिए भी किया जा सकता है। React में लिखा कोड सर्वर पर PHP, या अन्य क्लाइंट-साइड लाइब्रेरी जैसी किसी चीज़ के द्वारा प्रदान किए गए मार्कअप के साथ शांतिपूर्वक सहवास कर सकता है। वास्तव में, ठीक इसी तरहे फेसबुक में React का उपयोग किया जा रहा है।
 
-## ES6, ES2015, ES2016, etc {#es6-es2015-es2016-etc}
+## ES6, ES2015, ES2016, आदि {#es6-es2015-es2016-etc}
 
-These acronyms all refer to the most recent versions of the ECMAScript Language Specification standard, which the JavaScript language is an implementation of. The ES6 version (also known as ES2015) includes many additions to the previous versions such as: arrow functions, classes, template literals, `let` and `const` statements. You can learn more about specific versions [here](https://en.wikipedia.org/wiki/ECMAScript#Versions).
+ये सभी शब्द ECMAScript लैंग्वेज स्पेसिफिकेशन स्टैंडर्ड के सबसे हाल के संस्करणों को संदर्भित करते हैं, जो जावास्क्रिप्ट भाषा का एक कार्यान्वयन है। ES6 संस्करण (जिसे ES2015 के रूप में भी जाना जाता है) में पिछले संस्करणों में कई परिवर्धन शामिल हैं जैसे: तीर फ़ंक्शंस, क्लासेस, टेम्प्लेट शाब्दिक, `let` और `const` स्टेटमेंट। आप [यहां](https://en.wikipedia.org/wiki/ECMAScript#Versions) विशिष्ट संस्करणों के बारे में अधिक जान सकते हैं।
 
-## Compilers {#compilers}
+## कम्पाइलर {#compilers}
 
-A JavaScript compiler takes JavaScript code, transforms it and returns JavaScript code in a different format. The most common use case is to take ES6 syntax and transform it into syntax that older browsers are capable of interpreting. [Babel](https://babeljs.io/) is the compiler most commonly used with React.
+एक जावास्क्रिप्ट कंपाइलर जावास्क्रिप्ट कोड लेता है, और उसे बदल कर एक अलग प्रारूप में जावास्क्रिप्ट कोड देता है। सबसे आम उपयोग ES6 सिंटैक्स लेने और इसे ऊस सिंटैक्स में बदलने का है जिसे कि पुराने ब्राउज़र व्याख्या करने में सक्षम हैं। आमतौर पर React के साथ [बैबेल](https://babeljs.io/) कंपाइलर का उपयोग किया जाता है।
```

Comment:
`बैबेल` => `Babel`
I think we should not translate this

Reply:
> `बैबेल` => `Babel`
> I think we should not translate this

if we don't need to translate this may be we can add this to glossary for future reference

Reply:
Yes. Makes Sense. Let me add it right away